src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -116,6 +122,12 @@ def save_as_table(
             table_name if isinstance(table_name, str) else ".".join(table_name)
         )
         validate_object_name(full_table_name)
+        if column_order is None or column_order.lower() not in ("name", "index"):
```

Comment:
nit: `column_order is None` -> `not column_order`

Reply:
Sorry I didn't notice this comment and merged the PR.
But they are equivalent because `column_order.lower() not in ("name", "index")` will return True when `column_order` is "".

Reply:
Yeah I just feel `not column_order` is more pythonic (I got a lot of comments for this myself :p)